src/components/editor/document-bar/document-info/document-info-time-line.tsx

```diff
@@ -19,16 +20,16 @@ export enum DocumentInfoLineWithTimeMode {
   EDITED
 }
 
-export const DocumentInfoTimeLine: React.FC<DocumentInfoLineWithTimeProps> = ({ time, mode, userName, profileImageSrc }) => {
+export const DocumentInfoTimeLine: React.FC<DocumentInfoLineWithTimeProps> = ({ time, mode, userName, profileImageSrc, isInline }) => {
   useTranslation()
 
   const i18nKey = mode === DocumentInfoLineWithTimeMode.CREATED ? 'editor.modal.documentInfo.created' : 'editor.modal.documentInfo.edited'
   const icon: IconName = mode === DocumentInfoLineWithTimeMode.CREATED ? 'plus' : 'pencil'
 
   return (
-    <DocumentInfoLine icon={icon}>
+    <DocumentInfoLine icon={icon} isInline={isInline}>
       <Trans i18nKey={i18nKey} >
-        <UserAvatar photo={profileImageSrc} additionalClasses={'document-info-avatar font-style-normal bold font-weight-bold'} name={userName}/>
+        <UserAvatar photo={profileImageSrc} additionalClasses={`font-style-normal bold font-weight-bold ${!isInline ? 'document-info-avatar' : ''}`} name={userName}/>
```

Comment:
Here is this "isInline" again. The variable name doesn't say anything about what it does. How about  a "small/big user avatar" prop for the useravatar component instead?

Reply:
Ha, I knew there was something else with the isInline, but thought I removed that also.

I'll change it.

Reply:
Done in 54a91cb